js/libs/core.js

```diff
@@ -75,7 +75,17 @@ Fliplet.FormBuilder = (function() {
         };
       }
 
-      component.methods.setSourceValue = function(data) {
+      /**
+       * This function is responsible personalizating a default value when we need to get it from user personalized data
+       * For example: set default value to the user email
+       *
+       * @param {Object} data - an object with source and key properties
+       *  Where source - is defining the value  source type
+       *  And key - tell us what key in the source we should take as a value
+       *
+       * @return {void} - this function is not returns anything but updated value directlly with the help of the updateValue function.
```

Comment:
comment: this method does not return anything but updates the value directly on the target field

Reply:
@squallstar updated comment